examples/basic-mongo/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const connectMongo = require('connect-mongo');
+const session = require('express-session');
+const auth = require('@digipolis/auth');
+
+
+let {
```

Comment:
Waarom is dit een let?

Reply:
omwille van lijn 22-27, bij een const krijg je dan 
```
/home/stefan/astad/projects/auth_module_nodejs/examples/basic-mongo/index.js:23
sessionConfig = Object.assign(sessionConfig, {
              ^

TypeError: Assignment to constant variable.
    at Object.<anonymous> (/home/stefan/astad/projects/auth_module_nodejs/examples/basic-mongo/index.js:23:15)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3

```